src/components/studies/scheduler/Duration.tsx

```diff
@@ -127,12 +129,15 @@ const Duration: React.FunctionComponent<
         aria-label={unitLabel}
         {...props}
         value={unt}
+        label={placeHolder}
         sourceData={unitData}
         id={unitLabel.replace(' ', '')}
         onChange={e =>
           changeValue(num, e.target.value as moment.unitOfTime.Base)
         }
-        style={isIntro ? {width: '100px'} : undefined}></SelectWithEnum>
+        style={{width: '100px'}}>
```

Comment:
Instead of hard setting it, let's have it as an optional prop w/ a default value of 100px. That way we can reuse it and give it different width if we need to